### PR TITLE
UI changes according to new UX

### PR DESCRIPTION
**Pull request by sanketpathak**

### CODE REVIEW

File: src/app/shared/dependency-editor.service.ts [hunk 1]
```diff
@@ -113,6 +113,15 @@ export class DependencyEditorService {
...
+    getDependencyData2(url, payload): Observable < any > {
+    return this.http.get(url)
+        .map(this.extractData)
+        .map((data: StackReportModel | CveResponseModel | any) => {
+            return data;
+        })
+        .catch(this.handleError);
+    }
+
     getDependencyData1(url, payload): Observable < any > {
```
Comment (arunkumars08) [inline src/app/shared/dependency-editor.service.ts]: Please use proper function names.

File: src/app/shared/dependency-editor.service.ts [hunk 2]
```diff
@@ -113,6 +113,15 @@ export class DependencyEditorService {
             .catch(this.handleError);
     }
 
+    getDependencyData2(url, payload): Observable < any > {
```
Comment (arunkumars08) [inline src/app/shared/dependency-editor.service.ts]: Please use proper function names.

**Comment by arunkumars08** [comment]
@sanketpathak Please remove the image, it won't be used since the images is sent over API (data-img)

File: src/app/add-dependency/add-dependency.component.less [hunk 3]
```diff
@@ -406,13 +440,16 @@
...
             .list-view-pf-additional-info-item {
                 font-family: OpenSans;
                 font-size:12px;
                 color:#545b64;
-                width: 50%;
+                // width: 50%;
                 margin-right: 0;
+                display: -webkit-inline-box;
+                margin-left: -25px;
+                // width: 220px;
```
Comment (jyasveer) [inline src/app/add-dependency/add-dependency.component.less]: remove all the comments from the css file

File: src/app/add-dependency/add-dependency.component.ts [hunk 4]
```diff
@@ -142,16 +145,61 @@ export class AddDependencyComponent implements OnInit, OnDestroy, OnChanges {
       });
   }
 
+  changeTagname(master, t) {
```
Comment (jyasveer) [inline src/app/add-dependency/add-dependency.component.ts]: changeTagname can be changeTagName here

File: src/app/alert-box/alert-box.component.html [hunk 5]
```diff
@@ -57,11 +57,11 @@ <h2 class=" alert-title" style="color:#000">
       </div>
     </div>
   </div>
-  <span [style.background-color]="colored" style="/*background-color:#d30025#f8a33a;*/ padding: 12px 13px;color: #fff;/*position: relative;top: -140px;left: 238px;*/height: 40px;width: 40px;position: absolute;top: -45px;left: 248px;"
+  <span [style.background-color]="colored" style="/*background-color:#d30025#f8a33a;*/ padding: 12px 13px;color: #fff;height: 40px;width: 40px;position: absolute;top: 0px;left: 305px;"
     *ngIf="!isLoading && hasIssue !== 'na' && hasIssue">
     <i class="pficon-warning-triangle-o" aria-hidden="true" style="font-family:PatternFlyIcons-webfont;font-size:15px;color:#ffffff;"></i>
   </span>
-  <span style="/*background-color:#6ba100;*/ padding: 12px 13px;color: #fff;/*position: relative;top: -140px;left: 238px;*/height: 40px;width: 40px;position: absolute;top: -45px;left: 248px;"
+  <span style="/*background-color:#6ba100;*/ padding: 12px 13px;color: #fff;/*position: relative;top: -140px;left: 238px;*/height: 40px;width: 40px;position: absolute;top: 0px;left: 305px;"
```
Comment (jyasveer) [inline src/app/alert-box/alert-box.component.html]: could you please remove all the inline styles anywhere in the PR.

File: src/app/current-project/current-project.component.html [hunk 6]
```diff
@@ -13,7 +13,7 @@ <h2 class="card-pf-title" style="/*margin-left: -234px;margin-left: 10px;*/font-
           </h2>
           <div class="card-pf-body">
             <div class="card-pf-aggregate-status-notifications" style="font-size: 12px;/*width: 170px*/;margin-left: 32px;">
-              <p class="row" style="border: 1px solid;width:100%;height:58px;background:#ffffff;border:1px solid #bbbbbb;box-shadow:inset 0 2px 2px 0 rgba(0,0,0,0.08);height:72px;text-align: left;">
+              <p class="row" style="border: 1px solid;width:100%;height:58px;background:#f5f5f5;border:1px solid #bbbbbb;box-shadow:inset 0 2px 2px 0 rgba(0,0,0,0.08);height:72px;text-align: left;">
```
Comment (jyasveer) [inline src/app/current-project/current-project.component.html]: please move these inline styles to css file

File: src/app/dependency-editor/dependency-editor.component.ts [hunk 7]
```diff
@@ -224,10 +263,14 @@ export class DependencyEditorComponent implements OnInit, OnChanges {
...
       });
+      if (counter ++ > 4) {
+        alive = false;
+    }
+    });
   }
 
   private getCveData(payload: any) {
-    this.service.getDependencyData(this.getCveUrl, payload)
+    this.service.getDependencyData1(this.getCveUrl, payload)
```
Comment (jyasveer) [inline src/app/dependency-editor/dependency-editor.component.ts]: please give proper method names

File: src/app/license/license.component.ts [hunk 8]
```diff
@@ -51,7 +51,7 @@ export class LicenseComponent implements OnInit, OnChanges {
 
   constructor() {}
 
-  ngOnChanges() {
+  ngOnChanges() { this.stackStatus = '';
```
Comment (jyasveer) [inline src/app/license/license.component.ts]: please have a line break after the open brace

**Pull request closed without merging**